lib/pdf_fill/forms/va1010cg.rb

```diff
@@ -78,24 +78,11 @@ def subjects
 
       def merge_planned_facility_label_helper
         target_facility_code = @form_data.dig 'veteran', 'plannedClinic'
-
-        display_value = if Flipper.enabled? :caregiver_lookup_facility_name_db
-                          selected_facility = HealthFacility.find_by(station_number: target_facility_code)
-                          if selected_facility.nil?
-                            target_facility_code
-                          else
-                            "#{selected_facility.station_number} - #{selected_facility.name}"
-                          end
+        selected_facility = HealthFacility.find_by(station_number: target_facility_code)
+        display_value = if selected_facility.nil?
+                          target_facility_code
                         else
```

Comment:
🤔 Does the ternary wind up too long to inline here?
```suggestion
        display_value = selected_facility.nil? ? target_facility_code : "#{selected_facility.station_number} - #{selected_facility.name}"
```

Reply:
On second thought, it might not even be more readable.